src/main/java/com/netuitive/ananke/statsd/client/request/DecrementRequest.java

```diff
@@ -27,10 +27,15 @@ public DecrementRequest withTags(List<Tag> tags) {
         return this.tags == tags ? this : new DecrementRequest(this.metric, this.value, tags, this.sampleRate);
     }
 
-    public DecrementRequest withValue(Long value) {
+    public DecrementRequest withValue(Double value) {
```

Comment:
That was already done.  Line 30 has the old Long signature and Line 35 has the new Double signature for withValue().